File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/CourtPrice.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalTime

@Parcelize
data class CourtPrice(
    var id: String,
    var time: LocalTime,
    var price: Double,
    var courtClusterId: String
) : Parcelable

File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/CourtPrice.cs
namespace PickleballCourtBookingSystem.Api.Models


{
    public class CourtPrice
    {
        public Guid? Id { get; set; }
        public TimeSpan? Time { get; set; }
        public Double? Price { get; set; }
        public Guid? CourtClusterId { get; set; }
    }
}